models/machine.py

```diff
@@ -48,7 +49,13 @@ def create_benchmark_pipeline(self):
     def delete_benchmark_pipeline(self):
         buildkite.delete_pipeline(self.buildkite_pipeline_name)
 
-    def run_filters_and_skip_reason(self, benchmarkable_type, override_filters=None):
+    def run_filters_and_skip_reason(
```

Comment:
Not the scope, but oof this method name with no docstring is confusing. After reading I can see that "run" and "skip" are noun modifiers, not verbs, but that wouldn't be my first guess

Reply:
Yeah this codebase is really confusing. At the very least, type hints would help.

I'm not even sure the aforementioned `run_filters` are used anywhere except for validation during tests.

I'll add a quick docstring